build.fsx

```diff
@@ -29,7 +29,7 @@ let formattedRN =
 
 Target.create "Clean" (fun _ ->
     Shell.cleanDirs [ nupkgDir ]
-    Git.CommandHelper.directRunGitCommandAndFail "./Content" "clean -fxd"
+    // Git.CommandHelper.directRunGitCommandAndFail "./Content" "clean -fxd"
```

Comment:
Has this been an issue? 
I added that since there used to be a problem where I packaged up a lot of unnecessary files to the nupkg making it bigger than it should be.
I know this can be frustrating when adding a new file to the `./Content` dir, but otherwise it's a safety net for not pushing random stuff into package.

Reply:
Doh sorry. I didn't mean to commit that line - I'll revert it. But yes, that's why I commented it out - just temporarily. Good spot.